Gems/Atom/RHI/DX12/Code/Source/Platform/Windows/RHI/Device_Windows.cpp

```diff
@@ -614,7 +614,7 @@ namespace AZ
                         {
                             D3D12_AUTO_BREADCRUMB_OP op = currentNode->pCommandHistory[i];
 
-                            if (errorOccurred && i == actual + 1)
```

Comment:
Any idea what this magic +1 was doing here?

Reply:
I had ported the previous code which had a `<` comparison instead of a `<=`. The breadcrumbs being zero indexed should not have used a strict comparison before but I didn't catch it the first go around unfortunately.